torch_xla/csrc/torch_util.cpp

```diff
@@ -9,22 +9,22 @@
 namespace torch_xla {
 
 void SymIntElements::AddSymIntNodeElements(c10::SymInt& size) {
-  if (size.is_symbolic()) {
+  if (auto s = size.maybe_as_int()) {
```

Comment:
nit: do you need `.has_value()`?

Reply:
optional when used in a bool-y context will be true if it is full (even if its contents is zero) and false if empty. This is important since I can't do binding AND also test `has_value` for the conditional lol.